### PR TITLE
Updating record documenation with more details and fixing timeout values.

### PR DESCRIPTION
**Pull request by tituscha**
Updating record documentation with more details and fixing timeout values.

### CODE REVIEW

File: api-reference/beta/api/call-record.md [hunk 1]
```diff
@@ -144,6 +152,26 @@ Content-Type: application/json
 }
 ```
 
+##### Get recording file - Request
```
Comment (Lauragra) [inline api-reference/beta/api/call-record.md]: All examples must have the HTML comments to validate the request/response. Details here: https://msgo.azurewebsites.net/add/document/guidelines/auto-generate-and-author-content.html#html-comments

File: api-reference/beta/api/call-record.md [hunk 2]
```diff
@@ -10,7 +10,11 @@ ms.prod: "microsoft-teams"
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Record the call.
+Record a short audio clip from the call. This is useful if the bot wants to capture a voice response from the caller following a prompt.
```
Comment (Lauragra) [inline api-reference/beta/api/call-record.md]: Do the updates you're making here warrant an entry in the Changelog?

**Comment by Lauragra** [comment]
@VinodRavichandran , please review and sign off and set the do not merge or ready to merge label as appropriate.

File: api-reference/beta/api/call-record.md (hunk 1 above)
Comment (tituscha) [reply api-reference/beta/api/call-record.md]: Added HTML comments

File: api-reference/beta/api/call-record.md (hunk 2 above)
Comment (tituscha) [reply api-reference/beta/api/call-record.md]: No need. There is no change to the API we are just adding more details to it.

**Comment by Lauragra** [comment]
@tituscha , @VinodRavichandran , do you want to merge this PR? Or is it related to Paywu's PR #5276 ?

**Comment by tituscha** [comment]
> @tituscha , @VinodRavichandran , do you want to merge this PR? Or is it related to Paywu's PR #5276 ?

@Lauragra - Yes. This PR needs to be merged, this PR adds details to individual API. Paywu's PR is updating changes common across all our files and updates to the changelog.

**Pull request merged**